pkgs/by-name/ko/koboldcpp/package.nix

```diff
@@ -58,7 +58,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
 
   enableParallelBuilding = true;
 
-  LD_LIBRARY_PATH = [ addDriverRunpath.driverLink ];
+  LD_LIBRARY_PATH = [ ] ++ lib.optionals (config.cudaSupport) [ addDriverRunpath.driverLink ];
```

Comment:
Another note: this will be read by processes at build time (e.g. it'll make gcc look up files in this directory, and cmake, and so on); maybe use a distinguished name, or define something like `makeWrapperArgs`